source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,401 @@
 
-The auditor must force 50 Limit_Exceeded turnaways.
+Audit test requirements vary depending on the set up of the platform, as indicated by the Options within the tests below.
 
-* The auditor will log into the site and will navigate to and request an item. This will cause the user limit to reach the maximum allowance.The auditor will log into the site again using a different computer. The auditor will then repeat the action made on the previous computer (navigate to and request an item).
-* After the item has been requested the user should then be denied access. Each time access is refused, the auditor will record this as 1 Limit_Exceeded.
+Title Report audit tests for books and reference works apply to Host_Types Aggregated_Full_Content, eBook, and eBook_Collection.
 
-Each of these concurrent/simultaneous turnaways must report 1 Limit_Exceeded in the DR_D2 Standard View.
+Note that Data_Types Book and Reference_Work are included in the audit tests, as Reference_Works are included in the book-related Standard Views of the Title Report.
 
+E.2.4.1 Unique_Title_Investigations and Unique_Title_Requests
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-Audit Test D2-2: No_License
-...........................
+**Option 1**: Book_Segments and/or Reference_Items are available, users can elect to access Books and/or Reference_Works segment-by-segment.
 
-**IMPORTANT NOTE**: This test can only be carried out if the content provider restricts site content or if restricted content is not displayed. Any exclusion of tests must be confirmed by COUNTER prior to testing and the auditor notified.
+The auditor MUST request 100 Book_Segments and/or Reference_Items, 10 each from 10 different Books and/or Reference_Works. This MUST result in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 10 each of Unique_Title_Investigations and Unique_Title_Requests.
 
-The account used for this testing must have restricted access to content. The content the user has no license to access must be declared by the content provider prior to the testing. The auditor will attempt to access content using the account set up with restricted access. Each time access is refused, the auditor will record 1 No_License.
+**Option 2**: Only whole Books and/or Reference_Works are available.
 
-Each of these No_License turnaways must report 1 No_License in the DR_D2 Standard View.
+The auditor MUST request 20 Books and/or Reference_Works, twice each. This MUST result in 40 each of Total_Item_Investigations and Total_Item_Requests, and 20 each of Unique_Item_Investigations, Unique_Item_Requests, Unique_Title_Investigations and Unique_Title_Requests.
```

Comment:
This is correct if the Book_Segments/Reference_Items cannot be identified. If they can be identified, the result would be the same as for Option 3, which could be changed to also cover this case.